tasks/main.yml

```diff
@@ -35,6 +35,15 @@
     state: touch
     mode: 0644
 
+- name: Ensure that quarantine folder exists
+  ansible.builtin.file:
+    path: "{{ clamav_scan_quarantine_dir }}"
+    state: directory
+    owner: "{{ clamav_scan_quarantine_owner }}"
+    group: "{{ clamav_scan_quarantine_group }}"
+    mode: "{{ clamav_scan_quarantine_mode }}"
+  when: clamav_scan_move or clamav_scan_copy | bool
```

Comment:
More alphabetizing:
```suggestion
    group: "{{ clamav_scan_quarantine_group }}"
    mode: "{{ clamav_scan_quarantine_mode }}"
    owner: "{{ clamav_scan_quarantine_owner }}"
    path: "{{ clamav_scan_quarantine_dir }}"
    state: directory
  when: clamav_scan_copy or clamav_scan_move | bool
```

Reply:
I have doubts with this one. I would rather keep the order here. I find it is easier to read because:
- it makes more sense to have the module target as the first argument
- the module arguments are kept in the same order as the [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html), which is allegedly the most common

Reply:
Alright, I alphabetized both module arguments in the tasks and command flags in the script. I believe everything is ok now. Thanks for your review @dav3r!

Reply:
Thank you! 👍